addon/TelemetryRappor.jsm

```diff
@@ -318,44 +414,25 @@ var TelemetryRappor = {
    * @param {float} params.f - Value for probability f.
    * @param {float} params.p - Value for probability p.
    * @param {float} params.q - Value for probability q.
+   * @param {boolean} isSimulation - Boolean that indicates if the addon is run in a simulation.
+   * @param {int} simCohort - Cohort for the simulation.
```

Comment:
Should we call this simulatedCohort instead of simCohort?

Reply:
Done